### PR TITLE
[MINT-1868] Read view - DEA

### PR DESCRIPTION
**Pull request by patrickseguraoddball**
# MINT-1868

This PR is based on the work from an existing PR - https://github.com/CMS-Enterprise/mint-app/pull/1440

## Description

- Added translation config `pageLabel`  to add custom labels above read view questions
- Added translation config `hideIfFalsy` - ex:` dataWillNotBeCollectedFromParticipants`
- Updated translations a bit
- Added read view component
- Added unit test

## How to test this change

1.  Verify read view matches the figma from the ticket

## PR Author Checklist

- [ ] I have provided a detailed description of the changes in this PR.
- [ ] I have provided clear instructions on how to test the changes in this PR.
- [ ] I have updated tests or written new tests as appropriate in this PR.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.

### CODE REVIEW

File: src/features/ModelPlan/ChangeHistory/util.tsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ export type TranslationTables =
   | TableName.PLAN_OPS_EVAL_AND_LEARNING
   | TableName.PLAN_PAYMENTS
   | TableName.PLAN_COLLABORATOR
+  | TableName.PLAN_DATA_EXCHANGE_APPROACH
```
Comment (ClayBenson94) [inline src/features/ModelPlan/ChangeHistory/util.tsx:57]: Is this including some of the changes from https://github.com/CMS-Enterprise/mint-app/pull/1440 ?

Fine if so, just wanted to make sure I understood the scope of the diff here!

**Review by ClayBenson94 — APPROVED** [review]
I did some functional testing locally on this, and everything worked as I'd expect it to!

I'll wait for @garyjzhao to do a more in-depth review of the code!

(I think it might be worth merging https://github.com/CMS-Enterprise/mint-app/pull/1440 first since I think some changes from there are in here as well, might make this review simpler! I'll go take a look at it now!)

File: src/features/ModelPlan/ChangeHistory/util.tsx (hunk 1 above)
Comment (patrickseguraoddball) [reply src/features/ModelPlan/ChangeHistory/util.tsx:57]: Yep, I had a comment at the top of the PR noting this.

**Review by garyjzhao — CHANGES_REQUESTED** [review]
Mostly there. Missing two tooltips:

Figma:
image


Local:
image
image

**Review by garyjzhao — CHANGES_REQUESTED** [review]
The additional information is not showing up.

### Figma
image


### Local
image
image

**Pull request merged**